languages/python/oso/polar/host.py

```diff
@@ -76,7 +76,7 @@ def make_instance(self, name, args, kwargs, id):
         cls = self.get_class(name)
         try:
             instance = cls(*args, **kwargs)
-        except TypeError as e:
+        except Exception as e:
```

Comment:
Why did this exception type change?

Reply:
Because if python throws another error type, like a `ValueError` for example, it won't be caught and polar will crash. Try `x = new Integer("")`. As an aside, `x = new List(x)` will cause a segfault even with this change, but I haven't investigated why.

Reply:
This btw is the "improves Python error handling" change mentioned up top. Fwiw I think it makes the python library's behavior consistent with ruby.

Reply:
> As an aside, x = new List(x) will cause a segfault even with this change, but I haven't investigated why.

Ooof could you open a bug for this?

Reply:
Done